setup.py

```diff
@@ -2,14 +2,25 @@
 
 try:
     from setuptools import setup, Extension
+    from setuptools.command.test import test as TestCommand
 except ImportError:
-    print "You need setuptools to build this module"
+    print("You need setuptools to build this module.")
 
 from Cython.Build import cythonize
 
 import numpy as np #for the include dirs...
 import os, sys
 
+class PyTest(TestCommand):
+    def finalize_options(self):
+        TestCommand.finalize_options(self)
+        self.verbose = True
+
+    def run_tests(self):
+        import pytest
+        errno = pytest.main(self.test_args)
+        sys.exit(errno)
+
 
 include_dirs = [np.get_include(),
                 os.path.join('.', 'src')]
```

Comment:
we have this:
```
if sys.platform.startswith('win'):
    # need the stdint header for Windows (VS2008)
    include_dirs.append(os.path.join('.','src', 'win_headers'))
```
which should only be necessary for py2.7 (old Visual Studio). We're not set up to test on Windows with py3.4 or 3.5 -- so I'm wary of editing this. But it shoudl probably by something like:

sys.platform.startswith('win') and sys.version_info.major <= 2:
    include_dirs.append(os.path.join('.','src', 'win_headers'))